src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/nonassociations/NonAssociationsMigrationService.kt

```diff
@@ -69,22 +70,11 @@ class NonAssociationsMigrationService(
     with(context.body) {
       log.info("attempting to migrate $this")
 
-      if (isNotPrimaryNonAssociation()) {
-        telemetryClient.trackEvent(
```

Comment:
Not required, call to get nomis-prisoner-api determines primary associations only